README.md

```diff
@@ -133,27 +133,65 @@ $config = [
 Combining the previous steps will yield the following result:
 
 ```php
-add_filter('MyProject/ACF/fields/field_1', function ($field) {
+add_filter('MyProject/ACF/fields/field1', function ($field) {
   return [
     'label' => 'Sub Title',
-    'name' => 'sub_title',
+    'name' => 'subTitle',
     'type' => 'text'
-  ]
+  ];
 });
 
 add_filter('MyProject/ACF/locations/postTypePost', function ($location) {
   return [
     'param' => 'post_type',
     'operator' => '==',
     'value' => 'post'
+  ];
+});
+
+$config = [
+  'name' => 'group1',
+  'title' => 'My Group',
+  'fields' => [
+    'MyProject/ACF/fields/field1'
+  ],
+  'location' => [
+    [
+      'MyProject/ACF/locations/postTypePost'
+    ]
   ]
+];
+
+ACFComposer\ACFComposer::registerFieldGroup($config);
+```
+
+Executing this code will add a field with the **name** `subTitle` to all posts. The **key** will be a combination of the field group name, potentially all parent field names, and the field's name itself. In this case, that is `field_group1_subTitle`.
```

Comment:
- `subtitle`
- "potentially all parent field names" - makes me think "why potentially?" Am i missing an explanation there?
- "In this case, **this** is:"